common/placeholders.js

```diff
@@ -56,9 +68,22 @@ function getPlaceholders(types) {
   if (types.find(type => type === placeholderTypes.ATTACHMENT))
     placeholders = placeholders.concat(getAttachmentPlaceholders())
 
+  if (types.find(type => type === placeholderTypes.TABLE))
```

Comment:
Are you going to use the TABLE type? I'm not sure it's needed because it doesn't need to be in the placeholders menu.